pom.xml

```diff
@@ -0,0 +1,290 @@
+
+        <carbon.deployment.version>5.2.0</carbon.deployment.version>
+        <carbon.deployment.export.version>5.2.0</carbon.deployment.export.version>
+
+        <carbon.datasources.version>1.1.4</carbon.datasources.version>
+        <carbon.metrics.version>2.3.9</carbon.metrics.version>
+        <carbon.jndi.version>1.0.5</carbon.jndi.version>
+
+        <carbon.cache.version>1.1.3</carbon.cache.version>
+        <carbon.touchpoint.version>1.1.1</carbon.touchpoint.version>
+        <carbon.utils.version>2.0.8</carbon.utils.version>
+        <carbon.config.version>2.1.5</carbon.config.version>
+        <carbon.config.version.range>[2.1.5, 3.0.0)</carbon.config.version.range>
+        <carbon.securevault.version>5.0.11</carbon.securevault.version>
+
+        <slf4j.version>1.7.12</slf4j.version>
+        <slf4j.logging.package.import.version.range>[1.7.1, 2.0.0)
+        </slf4j.logging.package.import.version.range>
+
+        <!--<antlr4.runtime.version>4.5.1.wso2v1</antlr4.runtime.version>-->
+        <hikari.version>2.6.1</hikari.version>
+
+        <jaxen.import.version.range>[1.1.6,1.2.0)</jaxen.import.version.range>
+        <commons.logging.version>1.2</commons.logging.version>
+
+        <!--MSF4J related-->
+        <msf4j.version>2.6.4</msf4j.version>
+        <msf4j.import.version.range>[2.6.2, 3.0.0)</msf4j.import.version.range>
+        <com.fasterxml.jackson.core.version>2.9.6</com.fasterxml.jackson.core.version>
+        <com.fasterxml.jackson.core.version.range>[2.9.0, 2.10.0)</com.fasterxml.jackson.core.version.range>
+        <io.swagger.version>1.5.16</io.swagger.version>
+        <javax.ws.rs-api.version>2.0</javax.ws.rs-api.version>
+        <pax.logging.api.version>1.10.0</pax.logging.api.version>
+        <javax.servlet.version>3.1.0</javax.servlet.version>
+        <javax.websocket.api.version>1.1</javax.websocket.api.version>
+        <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
+
+        <equinox.osgi.version>3.11.0.v20160603-1336</equinox.osgi.version>
+        <pax.exam.version>4.7.0</pax.exam.version>
+        <pax.exam.testng.version>4.6.0</pax.exam.testng.version>
+        <pax.exam.link.mvn.version>4.6.0</pax.exam.link.mvn.version>
+        <pax.exam.container.native.version>4.6.0</pax.exam.container.native.version>
+        <pax.url.aether.version>1.6.0</pax.url.aether.version>
+        <org.jacoco.ant.version>0.7.9</org.jacoco.ant.version>
+        <jacoco.plugin.version>0.7.9</jacoco.plugin.version>
+        <ow2.jta.spec.version>1.0.12</ow2.jta.spec.version>
+        <commons.io.version>1.3.2</commons.io.version>
+        <osgi.test.util.version>5.1.1</osgi.test.util.version>
+        <slf4j.version>1.7.12</slf4j.version>
+
+        <bind.host.name>localhost</bind.host.name>
+
+        <log4j.version>1.2.17.wso2v1</log4j.version>
+        <testng.version>6.9.10</testng.version>
+        <mysql.connector.version>5.1.38</mysql.connector.version>
```

Comment:
@Tikiri16 shall we remove these since its not needed? please remove all the dependencies which uses these if not needed as well.

Reply:
Done.